### PR TITLE
refactor: createPost handler

### PR DESCRIPTION
**Pull request by Girishbari**
## Related Issue

**Refactored : handleSubmit handler in CreatePost**
## Description

earlier create post component would send toastify and resets the states even if the post was not able to save in the database, **made a negation in if condition** so that toastify and state would not able to reset if the problem has occurred during saving of post in DB

### CODE REVIEW

**Comment by IrfanshaikH-7** [comment]
Pakage lock !?😅 Yarn should've stoped it tho !

File: .yarnrc.yml
Comment (Sanchitbajaj02) [inline .yarnrc.yml:1]: remove this file

File: package-lock.json
Comment (Sanchitbajaj02) [inline package-lock.json:1]: remove this

**Review by Sanchitbajaj02 — CHANGES_REQUESTED** [review]

**Comment by Sanchitbajaj02** [comment]
@Girishbari Suggested some changes, please improve. Also create an issue first and linked that to this PR

**Review by Girishbari — COMMENTED** [review]
removed yarnrm.yml and package-lock.json

**Review by Sanchitbajaj02 — APPROVED** [review]

**Pull request merged**